packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -178,11 +176,8 @@ const resolveFixed = (image, options) => {
     )
   })
 
-  // Sort sizes for prettiness.
-  const sortedSizes = _.sortBy(filteredSizes)
```

Comment:
The array is initialized ordered. It is only pruned, not shuffled / added. So this sort should be a noop in any case.

Reply:
This warrants a comment that `fixedSizes` is expected to be sorted in the first place.

Reply:
Sure